index.html

```diff
@@ -64,33 +63,60 @@ <h2>
     <p>
       TBW
     </p>
+    <p>
+      TBW: <dfn>Example credential types</dfn>
+    </p>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="note" title="Definitions under discussion">
+The goal of the definitions in this section is to reuse or establish
+terminology that is common across a variety of digital credential formats and
+protocols. Discussions surrounding these definitions are active and
+the definitions are likely to change over the next several months.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically signed digital document containing one or more
+        [=claims=] made by an [=issuer=] about one or more [=subjects=].
+      </dd>
+      <dt>
+        <dfn>Credential manager</dfn>
+      </dt>
+      <dd>
+        Software that provides a user interface for querying, selecting, and
+        establishing consent for the [=presentation=] of a [=digital
+        credential=]. Examples include a digital wallet, or a login manager,
+        that manages various [=example credential types|credential types=].
+      </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential">query</dfn>
+      </dt>
+      <dd>
+        A format that a [=verifier=] uses, via an [=digital credential/exchange
+        protocol=], to request a [=digital credential=] from a [=credential
+        manager=].
       </dd>
       <dt>
-        <dfn>Identity credential provider</dfn>
+        <dfn>Presentation</dfn>
       </dt>
       <dd>
-        An application or service that provides a user interface for selecting
-        and/or querying a [=digital credential=], such as a digital wallet that
-        manages various identity documents and credentials.
+        A format that a [=credential manager=] uses, via an [=digital
```

Comment:
```suggestion
        A format that a [=holder=] uses, via an [=digital
```

Isn't the response produced by the holder?

Reply:
I guess, yes... I was thinking the holder was a human... but, you are right that the holder is just the software.

Reply:
Hrm, no, I'm with @marcoscaceres on this one. The "holder" is a role that a human can play. Note that it doesn't need to be a human, it can be any entity that has agency such as an organization or AI -- which we don't need to talk about in this spec 'cause it'll just confuse people.

That holder uses software to achieve their goals, where the software is a "credential manager/provider" AND a "user agent" (they use both via this API) to achieve a specific workflow.

IOW, the response is provided by software, so we need to name that software. I do think "credential manager/provider" is the right term here. Thoughts, @samuelgoto?

Reply:
Fixed in b080af4b378f32f2f9ae2add76698dc03652cab7 by separating the concept of a "holder" from "holder's software" and adding an example of "digital wallet" for "holder's software".